Blacksmithing and you: Practice working that chunk of metal until you know how to craft it into others chunks of metal!

Termineitor244 opened the pull request:
#### Summary
Content "Add practice recipes for learning how to be a proper blacksmith"

#### Purpose of change
We have suffered long enough in CDDA without proper practice recipes for blacksmithing! Those chunk of metals won't work themselves into something useful, and you shouldn't just have to try and craft hammers and nails until you somehow understand how this all works! What are, neanderthals?! We have books! You too can learn how to be a proper blacksmith with just time, money (materials) and a lot of nothing else to do! ;D

#### Describe the solution
Adds several practice recipes for all kinds of blacksmithing so you can actually try and learn how do do these things, and not just waste time and materials trying to craft whatever without stopping to understand what the hell you are even doing.

- [x]  Adds practice recipes for Redsmithing.
- [x]  Adds a new step for the Redsmithing proficiency, so you can just understand the basics first with things like copper, and then get better using the more rare and valuable bronze.
- [x] Adds the new Redsmithing proficiency to the recipes using bronze, you will take even longer to craft these if you don't first take the time to understand how to actually work soft metals.
- [x] Adds practice recipes for Blacksmithing.
- [x] Adds practice recipes for  Armorsmithing.
- [x] Adds practice recipes for Bladesmithing.
- [x] Adds practice recipes for Manual Tooling.
- [x] Adds practice recipes for Case Hardening.
- [x] Adds practice recipes for Quenching & Tempering.

Calculated values for material requirements per practice recipe (Formula being trying to craft 5 to 10 items of the relevant proficiency, adjusted by amount of work that could be done in the crafts in 1 hour):

<details> 
<summary>Redsmithing </summary>
Practice equivalent to making: 

- A crude copper knife (x1 copper, 25 min at the quickest without profiency).
- An spearhead for a copper spear/pike (2 coppers, 30 min/1 hour).
- An axe head from a copper hatchet (1 copper, 1 hour)

Those 3 basic forms give us: 4 pieces of copper for a 2 hour (Kinda) work, multiplied by 5: 20, divided in 2 for a one hour work: 10 pieces of copper for session.
</details>

<details> 
<summary> Advanced Redsmithing </summary>
Practice equivalent to making: 

- A (crude) bronze chisel (2 bronze, 45 min, reduced to 30 min for crudeness).
- A bronze mace head from a bronze mace (3 bronze, 3 hours).
- A bronze sword (4 bronze, 3 hours).
- A bronze helmet (7 bronze , 9 hours, 10 min, reduced to 4 hours for making a very crude bronze helmet that would not require armor smithing proficiency, you just go at it trying to make something resembling a helmet).

Those 4 complex forms give us: 16 bronze for 10 hours and 30 min of work, multiplied by 10: 160 bronze, divided by 10 for a one hour (and 3 min) work: 16 bronze per session.

Expanded the tools required to accommodate the different forms being crafted for practice.

</details>

<details> 
<summary> Blacksmithing </summary>
Practice equivalent to making: 

- A teapot or a steel bottle/tankard (1 steel standard, 1 hour and 30 min work as an average between the 3).
- A pair (one piece) of pliers (1 steel, 5 hours, reduced to 2 hours, 30 min to account for making it very crudely and not train manual tooling).
- A couple of steel wires (1 steel, 3 hours).
- A steel plate (4 steel, 9 hours).

Those 5 simple forms give us: 7 standard steel for 16 hours of work, multiplied by 10: 70 steel, divided by 16 for a one hour work: 4.375 (rounded up to 5) steel per session. Do take into account that the "steel" in use here is the steel_standard, which means if you work with steel chunks (Most likely) you require x4 the number of steel_standard in steel chunks, so 20 steel chunks per session.

</details>

<details> 
<summary> Armorsmithing </summary>
Practice equivalent to making: 

- A steel turban helmet (1 steel standard, 6 hour work).
- A steel light chestplate (4 steel, 10 days (240 hours) work, adjusted to 8 days (192 hours) for the couple of leather pieces used).
- A steel light leg guard (2 steel, 6 days, 12 hours (156 hours) work, adjusted to 5 days (120 hours) for the couple of leather pieces used).

Those 3 simple forms give us: 7 standard steel for 318 hours of work, multiplied by 10: 70 steel, divided by 318 for a one hour work: 0.22 steel per session. 
Absolutely too little, adjusted to making something like 30 armor pieces (x30), and making them very, very crudely (0.5 of the time) give us:  210 steel in 159 hours (rounded down to 150), divided by 150 for an one hour session: 1.4 steel (Rounded up to 2 steel), that is still little, so multiplied again x2 to, let's say make the player craft these armors bulkier to better understand how to make them (?), leave us with 4 standard steel for practice session.

One way to look at this is that the 1 hour sessions and the steel requirements of the recipes mean that you are basically practicing every hour making a very, very, very crude, quick and dirty chestplate (4 steel), or a couple of helmets and leg guards (again, 4 steel), testing their mobility/flexibility/use, discarding the steel monstrosities as scrap, and then starting again.

Do take into account that the "steel" in use here is the steel_standard, which means if you work with steel chunks (Most likely) you require x4 the number of steel_standard in steel chunks, so 16 steel chunks per session.

Reworked jacket requirement into a large cotton sheet for testing with the armor pieces.

</details>

<details> 
<summary> Bladesmithing </summary>
Practice equivalent to making: 

- A blade (1 steel standard (they actually require 1 chunk, but let's just assume you always break the first 3 you make), 1  hour).
- A metal sword (3 steel, 5 hours, let's leave it at 4 hours, 30 min for the carving of the wood in the original).
- A hatchet (1 steel, 5 hours)

These 3 simple but varied blade forms give us: 5 standard steel for 10 hours, 30 min of work, multiplied by 10: 50 steel, divided by 10 (let's omit the 30 min) for a one hour work:  5 steel per session. 

Or the equivalent to making and breaking 20 normal and small blades in one hour.

Do take into account that the "steel" in use here is the steel_standard, which means if you work with steel chunks (Most likely) you require x4 the number of steel_standard in steel chunks, so 20 steel chunks per session.

Expanded the tools required to accommodate the different forms being crafted for practice.

</details>

<details> 
<summary> Manual Tooling </summary>
Practice equivalent to making: 

-A drift (2 steel standard, 2 hours work).
-A lug wrench (2 steel standard (actually 7 steel chunks, but let's put them at the equivalent 8 chunks), 5 hours).
-A swage and die set (2 steel standard, 2 hours). 

These 3 complex but very useful forms give us: 6 standard steel for 9 hours of work, multiplied by 10: 60 steel, divided by 9 for a one hour work: 6.6 (rounded up to 7) steel per session. 

Do take into account that the "steel" in use here is the steel_standard, which means if you work with steel chunks (Most likely) you require x4 the number of steel_standard in steel chunks, so 28 steel chunks per session.

</details>

<details> 
<summary> Case Hardening </summary>
Practice equivalent to making: 

Hardened steel katana: 2 steel, 20 hours, even discounting the work into making the steel a proper blade, most of the time would still be spent just working the steel into the process of case hardening.

2 steel, x 10, in 20 hours, leaves us with 20 steel for 20 hours, at 1 hour of work per session that still results into 1 steel, I left it at 4 to account for the time in the original that you would spend making the steel into a proper blade, and to contemplate a lot, but I mean a whole lot, of steel craking and the like when you start in your first tries, the carbon is also already at x10 of what the original spends, and in 1 hour I seriously doubt you could make use of more.

One must take into account in these cases that the steel used is actually 4 steel chunks per steel_standard, and the carbon is 25 units per carbon requirement (So 250 carbon for one hour of work, a lot, but the amount and time of carbon should be the most important aspects of learning to work with case hardening).
</details>

<details> 
<summary> Quenching & Tempering </summary>
Practice equivalent to making: 

Tempered steel katana: It uses the same 2 steel, but takes even longer at 24 hours, same rationale as the last one.

Same as the last one, it uses 2 steel, for a 24 hour work, x10 is 20 steel, divided into 24 leave us with 0.8, even less than the last one, I leaved it at 4 too, there is just not that much time and what you need to learn is just the process of how much heat, for how much time, and how it changes the proprieties of the steel when working with this particular process.


</details>

#### Describe alternatives you've considered
Not giving love to Blacksmithing? I originally considered only doing Redsmithing, since that is what I'm currently interested, but I'm already there, so I wanted to finally give Blacksmithing some love, it seems like nobody else cares that you can't practice for these proficiencies and high level fabrication haha.

#### Testing
It works! The practice recipes appear and work as they should, and the new proficiency appears and has effects where it should too.

#### Additional context

Termineitor244 committed:
Redsmithing

Termineitor244 committed:
Redsmithing in mods

Termineitor244 committed:
Lint

Termineitor244 committed:
Blacksmithing

Termineitor244 committed:
Armorsmithing

Termineitor244 committed:
Bladesmithing

Termineitor244 committed:
Manual Tooling

Termineitor244 committed:
Case Hardening

Termineitor244 committed:
Quenching & Tempering

Termineitor244 committed:
Adjust numbers

RenechCDDA commented on data/json/recipes/practice/fabrication.json:
```diff
@@ -102,5 +102,239 @@
+    "time": "1 h",
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "tools": [ [ [ "surface_heat", 10, "LIST" ] ] ],
+    "components": [ [ [ "copper_scrap_equivalent", 2, "LIST" ] ] ],
```
Okay, so here's one immediate huge problem: This spends one hour and (essentially) 30mL of material to practice redsmithing? There is absolutely nothing useful someone can learn about working the materials without having the materials to work with... and the expected result should be that they waste a TON of materials learning.

You should need 100x more components than this, minimum.

There's also no reason it should be training fabrication. min_difficulty and skill_limit should be the same, I would say 4 or 5.

Termineitor244 replied:
This is based on the materials needed to craft a copper knife in the game (×2), I can probably ask for double what is in here right now, but asking for 10x times the material components is just asking way too much for what it is... Just an hour of work.

The rest of the practice recipes were made with other crafting recipes as baseline, some of them I can probably increase, maybe double the components, but you have to remember that these are one hour only endeavors, there is just not that much time, and yeah, you can definitely waste a lot of material training to become better at using that material, but there is no reason either to just waste large amounts of the materials working the fundamentals of the craft, all of these proficiencies will require the player to use these practice recipes several times, increasing the total materials used.

I could increase the time taken by each recipe, but all practice recipes are made with a standard 1 hour of time by default, and I don't think it would be good to change that for these.

About skill training... I don't really see the problem with these training your fabrication. You are using the materials to more or less craft until their breaking point things resembling actual projects you can do that do actually increase your fabrication in those levels, and you can't use these recipes to learn fabrication unless you already have a book telling you how to work in these kind of projects, but if that is a problem I could remove the skill learning from them, even though I don't really see the logic.

Termineitor244 replied:
That sounds reasonable. Hmmm...

Tell you what, I will give all of these a pass and increase the component requirements to 5x or 10x of the equivalent item craft, but I will round the materials down in accordance with the difference of time between the projects and the 1 hour base time for the practices, you can practice making 10 knifes or 5 knifes, but the time you have to do it is still 1 hour por session, so the materials should reflect that.

I will put this on draft until I'm done with it, and when I finish, I will publish a comment here/update the PR description saying what items I used for the materials needed, and in what number/how much of them/what % of such a project you could make in the 1 hour session.

RenechCDDA commented on data/json/recipes/practice/fabrication.json:
```diff
@@ -102,5 +102,239 @@
+      { "proficiency": "prof_redsmithing_adv", "time_multiplier": 1, "skill_penalty": 0 }
+    ],
+    "time": "1 h",
+    "components": [ [ [ "scrap_bronze", 4 ] ] ],
```
Same issue, needs to require **many** more.

TheShadowFerret commented on data/json/proficiencies/metalwork.json:
```diff
@@ -50,13 +50,25 @@
+    "id": "prof_redsmithing_adv",
+    "category": "prof_smithing",
+    "name": { "str": "Advanced Redsmithing" },
+    "description": "You can now easily work with the more ductile copper-containing metals and could probably make a living in the bronze age.",
```
```suggestion
    "description": "You can now easily work with the more ductile copper-containing metals and could probably make a living in the Bronze Age.",
```

Termineitor244 committed:
Update data/json/proficiencies/metalwork.json

Co-authored-by: TheShadowFerret <99621099+TheShadowFerret@users.noreply.github.com>

Termineitor244 commented:
Adjusted values per practice recipe:

<details> 
<summary>Redsmithing </summary>
Practice equivalent to making: 

- A crude copper knife (x1 copper, 25 min at the quickest without profiency).
- An spearhead for a copper spear/pike (2 coppers, 30 min/1 hour).
- An axe head from a copper hatchet (1 copper, 1 hour)

Those 3 basic forms give us: 4 pieces of copper for a 2 hour (Kinda) work, multiplied by 5: 20, divided in 2 for a one hour work: 10 pieces of copper for session.
</details>

<details> 
<summary>  </summary>
Practice equivalent to making: 

- 
- 
- 

</details>

<details> 
<summary>  </summary>
Practice equivalent to making: 

- 
- 
- 

</details>

<details> 
<summary>  </summary>
Practice equivalent to making: 

- 
- 
- 

</details>

<details> 
<summary>  </summary>
Practice equivalent to making: 

- 
- 
- 

</details>

<details> 
<summary>  </summary>
Practice equivalent to making: 

- 
- 
- 

</details>

<details> 
<summary>  </summary>
Practice equivalent to making: 

- 
- 
- 

</details>

<details> 
<summary>  </summary>
Practice equivalent to making: 

- 
- 
- 

</details>

<details> 
<summary>  </summary>
Practice equivalent to making: 

- 
- 
- 

</details>

Termineitor244 committed:
Redsmithing 2.0

Amount of copper equivalent to crafting:

-A crude copper knife (x1 copper, 25 min at the quickest without profiency).
-An spearhead for a copper spear/pike (2 coppers, 30 min/1 hour).
-An axe head from a copper hatchet (1 copper, 1 hour)
Those 3 basic forms give us: 4 pieces of copper for a 2 hour (Kinda) work, multiplied by 5: 20, divided in 2 for a one hour work: 10 pieces of copper for session.
Heat and byproducts adjusted too.
Reduced skill limit.

Termineitor244 committed:
Advanced Redsmithing 2.0

Amount of bronze equivalent to crafting:

-A (crude) bronze chisel (2 bronze, 45 min, reduced to 30 min for crudeness).
-A bronze mace head from a bronze mace (3 bronze, 3 hours).
-A bronze sword (4 bronze, 3 hours).
-A bronze helmet (7 bronze , 9 hours, 10 min, reduced to 4 hours for making a very crude bronze helmet that would not require armor smithing proficiency, you just go at it trying to make something resembling a helmet).

Those 4 complex forms give us: 16 bronze for 10 hours and 30 min of work, multiplied by 10: 160 bronze, divided by 10 for a one hour (and 3 min) work: 16 bronze per session.

Expanded the tools required to accommodate the different forms being crafted for practice.

Forge requirements and byproducts adjusted too.
Reduced skill limit.

Termineitor244 committed:
Blacksmithing 2.0

Amount of steel equivalent to crafting:

-A teapot or a steel bottle/tankard (1 steel standard, 1 hour and 30 min work as an average between the 3).
-A pair (one piece) of pliers (1 steel, 5 hours, reduced to 2 hours, 30 min to account for making it very crudely and not train manual tooling).
-A couple of steel wires (1 steel, 3 hours).
-A steel plate (4 steel, 9 hours).

Those 5 simple forms give us: 7 standard steel for 16 hours of work, multiplied by 10: 70 steel, divided by 16 for a one hour work: 4.375 (rounded up to 5) steel per session. Do take into account that the "steel" in use here is the steel_standard, which means if you work with steel chunks (Most likely) you require x4 the number of steel_standard in steel chunks, so 20 steel chunks per session.

Forge requirements and byproducts adjusted too.
Reduced skill limit.

Termineitor244 committed:
Armorsmithing 2.0

Amount of steel equivalent to crafting:

-A steel turban helmet (1 steel standard, 6 hour work).
-A steel light chestplate (4 steel, 10 days (240 hours) work, adjusted to 8 days (192 hours) for the couple of leather pieces used).
-A steel light leg guard (2 steel, 6 days, 12 hours (156 hours) work, adjusted to 5 days (120 hours) for the couple of leather pieces used).

Those 3 simple forms give us: 7 standard steel for 318 hours of work, multiplied by 10: 70 steel, divided by 318 for a one hour work: 0.22 steel per session.
Absolutely too little, adjusted to making something like 30 armor pieces (x30), and making them very, very crudely (0.5 of the time) give us:  210 steel in 159 hours (rounded down to 150), divided by 150 for an one hour session: 1.4 steel (Rounded up to 2 steel), that is still little, so multiplied again x2 to, let's say make the player craft these armors bulkier to better understand how to make them (?), leave us with 4 standard steel for practice session.

One way to look at this is that the 1 hour sessions and the steel requirements of the recipes mean that you are basically practicing every hour making a very, very, very crude, quick and dirty chestplate (4 steel), or a couple of helmets and leg guards (again, 4 steel), testing their mobility/flexibility/use, discarding the steel monstrosities as scrap, and then starting again.

Do take into account that the "steel" in use here is the steel_standard, which means if you work with steel chunks (Most likely) you require x4 the number of steel_standard in steel chunks, so 16 steel chunks per session.

Reworked jacket requirement into a large cotton sheet for testing with the armor pieces.
Forge requirements and byproducts adjusted too.
Reduced skill limit.

Termineitor244 committed:
Bladesmithing 2.0

Amount of steel equivalent to crafting:

-A blade (1 steel standard (they actually require 1 chunk, but let's just assume you always break the first 3 you make), 1  hour).
-A metal sword (3 steel, 5 hours, let's leave it at 4 hours, 30 min for the carving of the wood in the original).
-A hatchet (1 steel, 5 hours)

These 3 simple but varied blade forms give us: 5 standard steel for 10 hours, 30 min of work, multiplied by 10: 50 steel, divided by 10 (let's omit the 30 min) for a one hour work:  5 steel per session.

Or the equivalent to making and breaking 20 normal and small blades in one hour.

Do take into account that the "steel" in use here is the steel_standard, which means if you work with steel chunks (Most likely) you require x4 the number of steel_standard in steel chunks, so 20 steel chunks per session.

Expanded the tools required to accommodate the different forms being crafted for practice.

Forge requirements and byproducts adjusted too.
Reduced skill limit.

Termineitor244 committed:
Manual Tooling 2.0

Amount of steel equivalent to crafting:

-A drift (2 steel standard, 2 hours work).
-A lug wrench (2 steel standard (actually 7 steel chunks, but let's put them at the equivalent 8 chunks), 5 hours).
-A swage and die set (2 steel standard, 2 hours).

These 3 complex but very useful forms give us: 6 standard steel for 9 hours of work, multiplied by 10: 60 steel, divided by 9 for a one hour work: 6.6 (rounded up to 7) steel per session.

Do take into account that the "steel" in use here is the steel_standard, which means if you work with steel chunks (Most likely) you require x4 the number of steel_standard in steel chunks, so 28 steel chunks per session.

Forge requirements and byproducts adjusted too.
Reduced skill limit.

Termineitor244 commented:
I adjusted the requirements of almost all the practice recipes (Case Hardening and Quenching & Tempering pending), following the suggestion to make the player practice by trying to "craft" 5-10 times a basic item/form of the related proficiency but adjusted  by taking the time required for those crafts and the 1 hour sessions into account.

Adjusted values per practice recipe:

<details> 
<summary>Redsmithing </summary>
Practice equivalent to making: 

- A crude copper knife (x1 copper, 25 min at the quickest without profiency).
- An spearhead for a copper spear/pike (2 coppers, 30 min/1 hour).
- An axe head from a copper hatchet (1 copper, 1 hour)

Those 3 basic forms give us: 4 pieces of copper for a 2 hour (Kinda) work, multiplied by 5: 20, divided in 2 for a one hour work: 10 pieces of copper for session.
</details>

<details> 
<summary> Advanced Redsmithing </summary>
Practice equivalent to making: 

- A (crude) bronze chisel (2 bronze, 45 min, reduced to 30 min for crudeness).
- A bronze mace head from a bronze mace (3 bronze, 3 hours).
- A bronze sword (4 bronze, 3 hours).
- A bronze helmet (7 bronze , 9 hours, 10 min, reduced to 4 hours for making a very crude bronze helmet that would not require armor smithing proficiency, you just go at it trying to make something resembling a helmet).

Those 4 complex forms give us: 16 bronze for 10 hours and 30 min of work, multiplied by 10: 160 bronze, divided by 10 for a one hour (and 3 min) work: 16 bronze per session.

Expanded the tools required to accommodate the different forms being crafted for practice.

</details>

<details> 
<summary> Blacksmithing </summary>
Practice equivalent to making: 

- A teapot or a steel bottle/tankard (1 steel standard, 1 hour and 30 min work as an average between the 3).
- A pair (one piece) of pliers (1 steel, 5 hours, reduced to 2 hours, 30 min to account for making it very crudely and not train manual tooling).
- A couple of steel wires (1 steel, 3 hours).
- A steel plate (4 steel, 9 hours).

Those 5 simple forms give us: 7 standard steel for 16 hours of work, multiplied by 10: 70 steel, divided by 16 for a one hour work: 4.375 (rounded up to 5) steel per session. Do take into account that the "steel" in use here is the steel_standard, which means if you work with steel chunks (Most likely) you require x4 the number of steel_standard in steel chunks, so 20 steel chunks per session.

</details>

<details> 
<summary> Armorsmithing </summary>
Practice equivalent to making: 

- A steel turban helmet (1 steel standard, 6 hour work).
- A steel light chestplate (4 steel, 10 days (240 hours) work, adjusted to 8 days (192 hours) for the couple of leather pieces used).
- A steel light leg guard (2 steel, 6 days, 12 hours (156 hours) work, adjusted to 5 days (120 hours) for the couple of leather pieces used).

Those 3 simple forms give us: 7 standard steel for 318 hours of work, multiplied by 10: 70 steel, divided by 318 for a one hour work: 0.22 steel per session. 
Absolutely too little, adjusted to making something like 30 armor pieces (x30), and making them very, very crudely (0.5 of the time) give us:  210 steel in 159 hours (rounded down to 150), divided by 150 for an one hour session: 1.4 steel (Rounded up to 2 steel), that is still little, so multiplied again x2 to, let's say make the player craft these armors bulkier to better understand how to make them (?), leave us with 4 standard steel for practice session.

One way to look at this is that the 1 hour sessions and the steel requirements of the recipes mean that you are basically practicing every hour making a very, very, very crude, quick and dirty chestplate (4 steel), or a couple of helmets and leg guards (again, 4 steel), testing their mobility/flexibility/use, discarding the steel monstrosities as scrap, and then starting again.

Do take into account that the "steel" in use here is the steel_standard, which means if you work with steel chunks (Most likely) you require x4 the number of steel_standard in steel chunks, so 16 steel chunks per session.

Reworked jacket requirement into a large cotton sheet for testing with the armor pieces.

</details>

<details> 
<summary> Bladesmithing </summary>
Practice equivalent to making: 

- A blade (1 steel standard (they actually require 1 chunk, but let's just assume you always break the first 3 you make), 1  hour).
- A metal sword (3 steel, 5 hours, let's leave it at 4 hours, 30 min for the carving of the wood in the original).
- A hatchet (1 steel, 5 hours)

These 3 simple but varied blade forms give us: 5 standard steel for 10 hours, 30 min of work, multiplied by 10: 50 steel, divided by 10 (let's omit the 30 min) for a one hour work:  5 steel per session. 

Or the equivalent to making and breaking 20 normal and small blades in one hour.

Do take into account that the "steel" in use here is the steel_standard, which means if you work with steel chunks (Most likely) you require x4 the number of steel_standard in steel chunks, so 20 steel chunks per session.

Expanded the tools required to accommodate the different forms being crafted for practice.

</details>

<details> 
<summary> Manual Tooling </summary>
Practice equivalent to making: 

-A drift (2 steel standard, 2 hours work).
-A lug wrench (2 steel standard (actually 7 steel chunks, but let's put them at the equivalent 8 chunks), 5 hours).
-A swage and die set (2 steel standard, 2 hours). 

These 3 complex but very useful forms give us: 6 standard steel for 9 hours of work, multiplied by 10: 60 steel, divided by 9 for a one hour work: 6.6 (rounded up to 7) steel per session. 

Do take into account that the "steel" in use here is the steel_standard, which means if you work with steel chunks (Most likely) you require x4 the number of steel_standard in steel chunks, so 28 steel chunks per session.

</details>

.

Reduced maximum skill practice too.

I guess part of the expectation for a bigger number of steel required was because I'm using the steel_standard requirement, which is equivalent to 4 chunks for every steel_standard, so the numbers look smaller than they actually are.

And remember, this is precision work, it doesn't have to require gigantic quantities of steel if you need more detail in small pieces of work, even if made crudely, or working long times in a bigger piece.

TLDR: The numbers increased a little, but it seems that my initial estimate was mostly right and the practice recipes already required similar numbers to what you could accomplish trying to make 10 basic items of their respective proficiencies, taking into account the time needed to make even 1, and adjusting the materials needed considering your 1 hour work sessions. 

For example, the blade item requires only 1 steel chunk for 1 hour of work, so even if you make 10 shitty blades in one hour, you still only need 10 steel chunks, or if you make 20 blades (impossible, but let's say you just break most of them after minutes), you will still only need 20 chunks, or the equivalent of 5 steel_standard.

Termineitor244 committed:
Case Hardening 2.0

Hardened steel katana: 2 steel, 20 hours, even discounting the work into making the steel a proper blade, most of the time would still be spent just working the steel into the process of case hardening.

2 steel, x 10, in 20 hours, leaves us with 20 steel for 20 hours, at 1 hour of work per session that still results into 1 steel, I left it at 4 to account for the time in the original that you would spend making the steel into a proper blade, and to contemplate a lot, but I mean a whole lot, of steel craking and the like when you start in your first tries, the carbon is also already at x10 of what the original spends, and in 1 hour I seriously doubt you could make use of more.

This recipe was already spending a lot of material, I should rather be decreasing it...

One must take into account in these cases that the steel used is actually 4 steel chunks per steel_standard, and the carbon is 25 units per carbon requirement (So 250 carbon for one hour of work, a lot, but the amount and time of carbon should be the most important aspects of learning to work with case hardening).

Termineitor244 committed:
Quenching & Tempering 2.0

Tempered steel katana: It uses the same 2 steel, but takes even longer at 24 hours, same rationale as the last one.

Same as the last one, it uses 2 steel, for a 24 hour work, x10 is 20 steel, divided into 24 leave us with 0.8, even less than the last one, I leaved it at 4 too, there is just not that much time and what you need to learn is just the process of how much heat, for how much time, and how it changes the proprieties of the steel when working with this particular process.

Reduced the skill limit too.

Termineitor244 commented:
> This looks great, at a glance the numbers look good. It isn't a necessity but you might consider adding some of the details from your last few commits for the practice recipes to the steel crafting guide. https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/design-balance-lore/STEEL_CRAFTING.md

Thanks! Though I'm not really sure how to adapt what I wrote here to the actual steel crafting guide, I feel like my ramblings are not adequate haha.
